bot/exts/utilities/bookmark.py

```diff
@@ -105,6 +109,25 @@ def build_error_embed(message: str) -> discord.Embed:
             colour=Colours.soft_red,
         )
 
+    async def _delete_bookmark_context_menu_callback(
+        self,
+        interaction: discord.Interaction,
+        message: discord.Message,
+    ) -> None:
+        """
+        The callback that will be invoked upon using the "Delete Bookmark" context menu command.
+
+        This contains all the logic when it comes to deleting Sir-Lancebot's bookmark DM messages.
+        """
+        if interaction.channel.type != discord.ChannelType.private:
+            raise commands.UserInputError("You can only run this command your own DMs!")
+        elif message.channel != interaction.channel:
+            raise commands.UserInputError("You can only delete messages in your own DMs!")
+        elif message.author != self.bot.user:
+            raise commands.UserInputError("You can only delete messages sent by Sir Lancebot!")
+
+        await message.delete()
+
```

Comment:
After the message is deleted, I get the response below.
![image](https://github.com/python-discord/sir-lancebot/assets/15021300/bd3976f5-6480-43e7-a7ea-060774730bee)

Might just be me, but this could be worrying to a user, thinking the bot broke (I had thought this lol). If this is a convention for the rest of the bot (I haven't worked on this project in a bit), please let me know. In the meantime, here's a suggestion (probably best to put in a builder method like the others):

```suggestion
        await interaction.response.send_message(
            embed=discord.Embed(
                description="Bookmark successfully deleted.",
                colour=Colours.soft_green,
            ),
            ephemeral=True,
        )
```
        
        
![image](https://github.com/python-discord/sir-lancebot/assets/15021300/cbd67e48-faca-4699-aee8-d477b6ab24c6)